assets/src/scss/layout/_page-section.scss

```diff
@@ -1,4 +1,8 @@
-.page-section-header {
+.page-section-header _-- {
+  text-transform: none;
```

Comment:
This is customized by current themes on this class, but base didn't have the corresponding styles. Not sure if it really makes sense as a separate option :thinking: I'll check some live content to see how it looks like without.